picard/ui/metadatabox/tagdiff.py

```diff
@@ -241,39 +252,39 @@ def is_readonly(self, tag):
         """
         return bool(self.status[tag] & TagStatus.READONLY)
 
-    def add(self, tag, orig_values, new_values, removable, removed=False, readonly=False, top_tags=None):
+    def add(self, tag, old=None, new=None, removable=True, removed=False, readonly=False, top_tags=None):
         """
         Adds tag information to the TagDiff and updates its status.
 
         Args:
             tag: The tag name (string).
-            orig_values: The original tag value(s).
-            new_values: The new tag value(s).
+            old: The old tag value(s).
+            new: The new tag value(s).
             removable (bool): Whether the tag can be removed.
             removed (bool): Whether the tag was marked as removed.
             readonly (bool): Whether the tag is read-only.
             top_tags (set): Set of top level tags
         """
-        if orig_values:
-            self.orig.add(tag, orig_values)
+        if old:
```

Comment:
We call it "original" elsewhere in the code base, and also in the UI. I would stick with that.

Reply:
"old"/"new" are very common when it comes to "diff".
I agree we use orig/original in the rest of the code, but in the context of TagDiff it doesn't make much sense because we can compare any set of tags (not only "original tags" vs "new tags").
For any coder, old to new direction is obvious, while orig to new is only obvious for one familiar with Picard code and the meaning of "orig".

Reply:
Ok, that's convincing